python/mxnet/module/module.py

```diff
@@ -73,6 +75,11 @@ def __init__(self, symbol, data_names=('data',), label_names=('softmax_label',),
         assert len(work_load_list) == len(self._context)
         self._work_load_list = work_load_list
 
+        if group2ctxs is None:
```

Comment:
This seems redundant since you already checked `None` in DataExecutorGroup?

Reply:
Some user may use `DataExecutorGroup` directly. And there are such cases in `test_shared_exec_group` in `test_module.py`.

Reply:
Is the same check in module not necessary then?

Reply:
Right, removed.